### PR TITLE
[ISSUE #380]Translation for "数据集成-RocketMQ Connect实战1" Section in the v4.x & 5.0 Document (CN -> EN) 

### LINKED ISSUE #380: Translation for "数据集成-RocketMQ Connect实战1" Section in the v4.x & 5.0 Document (CN -> EN)
The Chinese version of the "数据集成-RocketMQ Connect实战1" section in the v4.x & v5.0 document has now been finished. However, its English version is still incomplete.

We now require someone who knows both Chinese and English and understands the technical terms of Apache RocketMQ to build it. The translation work is not only copy/paste from translation tools but also requires you to have some basic knowledge of Apache RocketMQ and technical writing.

### PR DESCRIPTION
**Pull request by mxsm**
Please do not create a Pull Request without creating an issue first. 

## What is the purpose of the change

close #380 

## Brief changelog

- Translation for "数据集成-RocketMQ Connect实战1" Section in the v4.x & 5.0 Document (CN -> EN) 

## Verifying this change

XXXX

Follow this checklist to help us incorporate your contribution quickly and easily:

- [x] Make sure there is a Github issue filed for the change (usually before you start working on it). Trivial changes like typos do not require a Github issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue. 
- [ ] Format the pull request title like `[ISSUE #123] Fix UnknownException when host config not exist`. Each commit in the pull request should have a meaningful subject line and body.
- [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
- [ ] Write necessary unit-test to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add integration-test in [test module](https://github.com/apache/rocketmq/tree/master/test).
- [ ] Run `mvn -B clean apache-rat:check findbugs:findbugs checkstyle:checkstyle` to make sure basic checks pass. Run `mvn clean install -DskipITs` to make sure unit-test pass. Run `mvn clean test-compile failsafe:integration-test`  to make sure integration-test pass.
- [ ] If this contribution is large, please file an [Apache Individual Contributor License Agreement](http://www.apache.org/licenses/#clas).

### CODE REVIEW

File: i18n/en/docusaurus-plugin-content-docs/current/08-数据集成/28RocketMQ Connect In Action1.md [hunk 1]
```diff
@@ -1,62 +1,65 @@
-# RocketMQ Connect实战1
+# RocketMQ Connect actual combat 1
```
Comment (tsunghanjacktsai) [inline i18n/en/docusaurus-plugin-content-docs/current/08-数据集成/28RocketMQ Connect In Action1.md]: RocketMQ Connect in Action 1

File: i18n/en/docusaurus-plugin-content-docs/current/08-数据集成/28RocketMQ Connect In Action1.md [hunk 2]
```diff
@@ -1,62 +1,65 @@
...
 
 MySQL Source(CDC)  -  >RocketMQ Connect  ->  MySQL Sink(JDBC)
 
-## 准备
+## Prepare
```
Comment (tsunghanjacktsai) [inline i18n/en/docusaurus-plugin-content-docs/current/08-数据集成/28RocketMQ Connect In Action1.md]: Preparation

File: i18n/en/docusaurus-plugin-content-docs/current/08-数据集成/28RocketMQ Connect In Action1.md [hunk 3]
```diff
@@ -1,62 +1,65 @@
...
 Debezium RocketMQ Connector
+
 ```
 $ cd rocketmq-connect/connectors/rocketmq-connect-debezium/
 $ mvn clean package -Dmaven.test.skip=true
 ```
 
-将 Debezium MySQL RocketMQ Connector 编译好的包放入Runtime加载目录。命令如下：
-```
+Put the compiled Debezium MySQL RocketMQ Connector package into the Runtime loading directory. The command is as follows：
```
Comment (tsunghanjacktsai) [inline i18n/en/docusaurus-plugin-content-docs/current/08-数据集成/28RocketMQ Connect In Action1.md]: Put -> Move

File: i18n/en/docusaurus-plugin-content-docs/current/08-数据集成/28RocketMQ Connect In Action1.md [hunk 4]
```diff
@@ -1,62 +1,65 @@
...
+
+```shell
 mkdir -p /usr/local/connector-plugins
 cp rocketmq-connect-debezium-mysql/target/rocketmq-connect-debezium-mysql-0.0.1-SNAPSHOT-jar-with-dependencies.jar /usr/local/connector-plugins
 ```
 
 JDBC Connector
 
-将 JDBC Connector 编译好的包放入Runtime加载目录。命令如下：
+Put the compiled JDBC Connector package into the Runtime loading directory. The command is as follows：
```
Comment (tsunghanjacktsai) [inline i18n/en/docusaurus-plugin-content-docs/current/08-数据集成/28RocketMQ Connect In Action1.md]: Put -> Move

File: i18n/en/docusaurus-plugin-content-docs/current/08-数据集成/28RocketMQ Connect In Action1.md [hunk 5]
```diff
@@ -199,16 +206,16 @@ curl -X POST -H "Content-Type: application/json" http://127.0.0.1:8082/connector
 }'
 ```
 
-### 启动 jdbc sink connector
+### Start jdbc sink connector
```
Comment (tsunghanjacktsai) [inline i18n/en/docusaurus-plugin-content-docs/current/08-数据集成/28RocketMQ Connect In Action1.md]: jdbc -> JDBC

**Review by tsunghanjacktsai — CHANGES_REQUESTED** [review]
Hey @mxsm ,

Well done, only some minor changes should be made. Don't forget to correct the corresponding changes in the 5.0 version.

**Comment by mxsm** [comment]
@tsunghanjacktsai  PTAL~

**Review by tsunghanjacktsai — APPROVED** [review]

**Review by RongtongJin — APPROVED** [review]

**Pull request merged**